ReactNativeClient/lib/models/Setting.js

```diff
@@ -552,7 +563,15 @@ class Setting extends BaseModel {
 				label: () => _('Ignore TLS certificate errors'),
 			},
 
-			'sync.wipeOutFailSafe': { value: true, type: Setting.TYPE_BOOL, advanced: true, public: true, section: 'sync', label: () => _('Fail-safe: Do not wipe out local data when sync target is empty (often the result of a misconfiguration or bug)') },
+			'sync.wipeOutFailSafe': {
+				value: true,
+				type: Setting.TYPE_BOOL,
+				advanced: true,
+				public: true,
+				section: 'sync',
+				label: () => _('Fail-safe'),
+				description: () => _('Do not wipe out local data when sync target is empty (often the result of a misconfiguration or bug)'),
+			},
```

Comment:
Please set the description to "Fail-safe: Do not wipe out local data when sync target is empty (often the result of a misconfiguration or bug)", otherwise it will invalidate all the translations which I don't think is necessary.

Reply:
Good catch, thanks!